### DISCUSSION TITLE
Response status 302 ,Location don't match request url

### DISCUSSION THREAD

**Discussion by lulu-0126** [Kong/kong#7235] [Help] [answered]
### Summary

Use kong to do a request proxy, the correct request result returns a 302 redirect, Except Location is all the URL of the current request, but the actual returned Location lacks port and url prefix

### Additional Details & Logs

- Kong version 1.1.1
- Kong debug-level startup logs (`$ kong start --vv`)
curl -g -i -X GET http://ip:8000/nova/v2.1 -H "Accept: application/json" -H "User-Agent: python-novaclient" -H "X-Auth-Token”

Content-Type: text/plain; charset=utf8
Content-Length: 0
Connection: keep-alive
Location: http://ip/v2.1/   ----expect http://ip:8000/nova/v2.1
X-Openstack-Request-Id: req-e3856560-762f-4ec7-967c-21b62b5ae0e3
X-Compute-Request-Id: req-e3856560-762f-4ec7-967c-21b62b5ae0e3
Date: Thu, 22 Apr 2021 09:50:55 GMT
X-Kong-Upstream-Latency: 203
X-Kong-Proxy-Latency: 1
Via: kong/1.1.1

- Kong configuration (the output of a GET request to Kong's Admin port - see
  https://docs.konghq.com/latest/admin-api/#retrieve-node-information)
- centos7

**Comment by bungle** [comment, marked as answer]
The response and the Location header seems to come from upstream. Kong does not touch it. But if you want you can write a plugin to adjust that on `header_filter` phase. Did I understood it correctly?
